crates/virtio-queue-ser/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# v0.4.0
+
+## Added 
+
+- Derived `Eq` for `state::QueueStateSer`.
+
+## Changed
```

Comment:
We should mention about the Rust edition update.

Reply:
done!